srcpkgs/vkBasalt/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'vkBasalt'
+pkgname=vkBasalt
+version=0.3.2.1
+revision=1
+archs="x86_64 i686"
```

Comment:
Is there anything that would it make it not work on ARM or PPC? Or `x86_64-musl`, even?

Reply:
as far as i can tell, there isn't